### PR TITLE
Create CrossWeave.Protocol.toml

### PR DESCRIPTION
**Pull request by NestorLtd**
CrossWeave.Protocol.toml

### CODE REVIEW

**Review by eherrerosj — CHANGES_REQUESTED** [review]
the link added to orgs is a user link not an org link. Please keep the repo only and add the ecosytem name to the sub_ecosystems list of the compatible ecosystem(s) toml (s)

**Comment by NestorLtd** [comment]
Ive done what you asked. Enjoy your Christmas time, see you all next year.

**Comment by eherrerosj** [comment]
> Ive done what you asked. Enjoy your Christmas time, see you all next year.

I don't see it be part of any other ecosystem

File: data/ecosystems/c/CrossWeave.Protocol.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Ecosystem Level Information
+title = "CrossWeave Protocol"
+
+sub_ecosystems = []
+
+github_organizations = ["https://github.com/NestorLtd"]
```
Comment (jubos) [inline data/ecosystems/c/CrossWeave.Protocol.toml:6]: This isn't an organization, it's a user.  Please remove it.

**Review by jubos — CHANGES_REQUESTED** [review]

**Comment by jubos** [comment]
Hi, please address the changes and reopen when it's ready. Thanks.

**Pull request closed without merging**